catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -182,15 +191,24 @@ public CreateResponse create(
 
       createResponse = doCreate(createRequest);
     } catch (IngestException e) {
+      INGEST_LOGGER.debug("Error ingesting resource: {}.", fileName, e);
```

Comment:
✏️ I don't think we need this debug statement if we're doing the info log in the finally block.

Reply:
fixed